server.js

```diff
@@ -5,10 +5,8 @@ import WebpackDevServer from 'webpack-dev-server';
 import config from './webpack.local.config';
 
 new WebpackDevServer(webpack(config), {
-  publicPath: config.output.publicPath,
```

Comment:
Won't this break something?

Reply:
I tested locally and everything worked as expected, and this is just a development server